### ISSUE TITLE
Request for Reactjs + Aspnetcore web api OBO

### ISSUE THREAD

**Issue by franva** [Azure-Samples/ms-identity-javascript-react-spa-dotnetcore-webapi-obo#1]
Many of the existing examples of SPA projects are either out of date, or not update to date.

Reactjs is a very popular front-end lib and I think it deserves some love. I did see Angular, but Reactjs is more popular then Angular, so if there is one example for Angular, I think there should also be one example for Reactjs.

Also, just wondering, when will this project be available?

**Comment by derisen** [comment]
hi @franva -you can see the samples we maintain [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code). You're right about React and we are working on it. The PR for this sample should be open in a few days.

**Comment by franva** [comment]
thanks @derisen I found that place before, I used "Calls own web API" but the example project doesn't work in Firefox, only works in Edge. Looking forward the upcoming Reactjs project. 

I hope to see 2 types of reactjs examples:

1. Reactjs + Asp.Net Core Web Api to fetch emails from Hotmail. The Reactjs app and WebApi live in different places, so this is OBO flow

2. Reactjs as front-end + Asp.Net Core Web Api to fetch emails from Hotmail. The Reactjs and WebApi live in ONE Project and is hosted together on same server.

I am looking forward to the upcoming examples.

Appreciate the efforts MS put on this project.
